support/k8s/k8s-workload-registrar/config.go

```diff
@@ -36,18 +36,20 @@ type Mode interface {
 }
 
 type CommonMode struct {
-	LogFormat          string   `hcl:"log_format"`
-	LogLevel           string   `hcl:"log_level"`
-	LogPath            string   `hcl:"log_path"`
-	TrustDomain        string   `hcl:"trust_domain"`
-	ServerSocketPath   string   `hcl:"server_socket_path"`
-	AgentSocketPath    string   `hcl:"agent_socket_path"`
-	ServerAddress      string   `hcl:"server_address"`
-	Cluster            string   `hcl:"cluster"`
-	PodLabel           string   `hcl:"pod_label"`
-	PodAnnotation      string   `hcl:"pod_annotation"`
-	Mode               string   `hcl:"mode"`
-	DisabledNamespaces []string `hcl:"disabled_namespaces"`
+	LogFormat          string            `hcl:"log_format"`
+	LogLevel           string            `hcl:"log_level"`
+	LogPath            string            `hcl:"log_path"`
+	TrustDomain        string            `hcl:"trust_domain"`
+	ServerSocketPath   string            `hcl:"server_socket_path"`
+	AgentSocketPath    string            `hcl:"agent_socket_path"`
+	ServerAddress      string            `hcl:"server_address"`
+	Cluster            string            `hcl:"cluster"`
+	PodLabel           string            `hcl:"pod_label"`
+	PodAnnotation      string            `hcl:"pod_annotation"`
+	Mode               string            `hcl:"mode"`
+	DisabledNamespaces []string          `hcl:"disabled_namespaces"`
+	IdentityTemplate   string            `hcl:"identity_template"`
+	Context            map[string]string `hcl:"context"`
```

Comment:
Should these be defined in config_crd.go because they are specific to the crd mode?

Reply:
If we want to support this feature only on CRD mode, as it was showcased here, then yes.